docs/chapters/infrastructure/ci-cd/build-slaves.rst

```diff
@@ -0,0 +1,56 @@
+Build Slaves
+============
+
+Running a public slave
+----------------------
+
+In order to run a slave which the master can reach via SSH you still need to prepare some things:
+
+.. code-block:: bash
+   
+   sudo apt install openjdk-8-jdk-headless
+   sudo apt install docker.io
+   sudo apt install vagrant
+   sudo useradd jenkins
+   sudo usermod -aG docker jenkins
+
+Running a private slave
+-----------------------
+
+Sometimes it's necessary to run slaves from a private network. Then you need to run a java program which will connect to the master instead the master connecting to the slave. To do so, set up a now slave in Jenkins.
+
+Prerequesits
+____________
+
+You need to have the same Java version running on the slave as your Jenkins master runs. Nowadays the docker LTS version runs Java 8.
+
+Best practice is also to create a unix user the slave will run with on your machine, you can call the user jenkins. If you will run docker, make sure this user is in the docker group.
```

Comment:
Should we write these docs more like "do this" rather than allowing too many options and considerations creep into the instructions? For example, if we believe it's best practice to do something, should we just describe how to do that then? I think it's a good idea to have a rationale, and sometimes even compare options, but maybe we should keep that at a minimum.

A nitpick: if we mean "going to use Docker in the setup" then "if you will run docker" might be ambiguous since it could also mean just the act of starting docker, not the concept of using docker as a part of the setup.